<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Students</class>
 <widget class="QMainWindow" name="Edit_Students">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 34, 34);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font:16pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLabel:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Edit Student Profile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cLast_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cFirst_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cMiddle_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cID_number">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cYearLevel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cGender">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cCourse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 9pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 9pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>106</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 9pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 9pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>ID Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>86</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 9pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Year Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>60</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 9pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 9pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Course</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
